src/test/rpc/AccountTx_test.cpp

```diff
@@ -244,6 +248,80 @@ class AccountTx_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testParametersApiV2()
```

Comment:
Might wanna:
- Add a parameter for api_version.
- Call this method in a loop over versions from apiMinimumSupportedVersion to apiBetaVersion.
- Remove the other method.
- Make the assertion on the error value conditional on the api_version parameter, using condition < 2 for the old behavior and else for the new behavior.

Reply:
https://github.com/XRPLF/rippled/pull/4552#discussion_r1242329180